elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/AggregationDataStoreTestHarness.java

```diff
@@ -11,31 +11,48 @@
 import com.yahoo.elide.datastores.aggregation.core.NoopQueryLogger;
 import com.yahoo.elide.datastores.aggregation.metadata.MetaDataStore;
 import com.yahoo.elide.datastores.aggregation.queryengines.sql.SQLQueryEngine;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.SQLDialectFactory;
 import com.yahoo.elide.datastores.jpa.JpaDataStore;
 import com.yahoo.elide.datastores.jpa.transaction.NonJtaTransaction;
 import com.yahoo.elide.datastores.multiplex.MultiplexManager;
 
 import org.hibernate.Session;
 
+import lombok.AllArgsConstructor;
+
+import java.util.Collections;
+import java.util.Map;
 import java.util.function.Consumer;
 
 import javax.persistence.EntityManager;
 import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
 
+@AllArgsConstructor
 public class AggregationDataStoreTestHarness implements DataStoreTestHarness {
     private EntityManagerFactory entityManagerFactory;
+    private DataSource defaultDataSource;
```

Comment:
Why do we still need an entityManagerFactory if we are using Hikari and JDBC directly?

Reply:
entityManagerFactory is still required for JPA datastore.